pkg/auth/handler/webapp/authflowv2/account_status.go

```diff
@@ -0,0 +1,35 @@
+package authflowv2
+
+import (
+	"net/http"
+
+	handlerwebapp "github.com/authgear/authgear-server/pkg/auth/handler/webapp"
+	"github.com/authgear/authgear-server/pkg/auth/handler/webapp/viewmodels"
+	"github.com/authgear/authgear-server/pkg/util/httproute"
+	"github.com/authgear/authgear-server/pkg/util/template"
+)
+
+var TemplateWebAuthflowAccountStatusHTML = template.RegisterHTML(
```

Comment:
It seems this pr has a conflict with the issue which davis is working on
https://github.com/authgear/authgear-server/issues/3582